web/.storybook/main.cjs

```diff
@@ -1,18 +1,20 @@
 const path = require('path');
 module.exports = {
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```

Comment:
Do we want to support non ".stories.EXT" files?

Reply:
Also don't know why the two entries are not merged together 🤔

Reply:
Apparently they deprecated that according to the migration 🤷🏻‍♂️